app/models/post.rb

```diff
@@ -0,0 +1,23 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  def five_recent_comments
+    comments.order('created_at Desc').limit(5)
+  end
+
+  def update_post_counter
+    author.increment!(:posts_counter)
+  end
+
+  private
+
+  def update_comments_count
+    update(CommentsCounter: comments.count)
+  end
```

Comment:
- [ ] @PNdunguMaina great work in your methods, however this method is suppose to be in the comments model, kindly move it to the comment model.